### DISCUSSION TITLE
JATS-to-PDF pipeline using only HTML (with CSS for print) and not using LaTeX

### DISCUSSION THREAD

**Discussion by castedo** [singlesourcepub/community#50] [Announcements]
During the Nov community call, I mentioned progress on a JATS-to-PDF pipeline that does not rely on LaTeX to produce PDFs. It is a pipeline producing HTML with CSS for print which is then "eprinted" to a PDF. With the help of courtbouillon.org the pipeline is producing significantly higher quality PDFs.

You can see PDF examples and details on how they are generated at http://ref.castedo.com/epijats/

**Comment by xldrkp** [comment]
Great! Do you want to report on that in todays call?

**Reply by xldrkp** [reply]
For everybody interested please find the links from the session in our pad at https://writemd.rz.tuhh.de/gEJliKV5TQO1_AWmtGDIQg?both#2022-12-05--12-am

**Comment by kaveh1000** [comment]
Pls also look at other discussions on this
<https://github.com/singlesourcepub/community/discussions> page. I had not
come across these guys before. We need to take this, PagedJS, etc into
consideration...

On Mon, 5 Dec 2022 at 11:33, Axel Dürkop ***@***.***> wrote:

> For everybody interested please find the links from the session in our pad
> at http://ref.castedo.com/epijats/
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/singlesourcepub/community/discussions/50#discussioncomment-4312352>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABBPDK3BPSCGZKAIQARSEFDWLXHJJANCNFSM6AAAAAASTX4JIQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***
> .com>
>


-- 
Kaveh Bazargan PhD
Director
River Valley Technologies <http://rivervalley.io> ● Twitter
<https://twitter.com/rivervalley1000> ● LinkedIn
<https://www.linkedin.com/in/bazargankaveh/> ● ORCID
<https://orcid.org/0000-0002-1414-9098> ● @***@***.***
***@***.***>
*Accelerating the Communication of Research*
